src/libraries/Microsoft.Extensions.Configuration.Binder/tests/SourceGenerationTests/Baselines/net462/ServiceCollection/Configure_T.generated.txt

```diff
@@ -119,6 +119,10 @@ namespace Microsoft.Extensions.Configuration.Binder.SourceGeneration
         {
             foreach (IConfigurationSection section in configuration.GetChildren())
             {
+                if (!string.IsNullOrEmpty(section.Value) && !section.GetChildren().GetEnumerator().MoveNext())
```

Comment:
Couldn't this be generated using `!section.GetChildren().Any()` instead? It could theoretically avoid the enumerator allocation provided that the underlying enumerable implements `ICollection<T>`.

Reply:
This will need to add a dependency on System.Linq which comes with its own complication if the project not referencing it.

Reply:
There are plenty of existing spots in this generated code that might benefit from Linq, or even this API: https://github.com/dotnet/runtime/issues/83692

Not sure we should tackle introducing new dependencies to this code at this point for a regression bugfix.  Instead we should think about minimizing this change and reducing risk as we'll likely want to backport it.  Codebase refactoring can be captured in a separate issue.

Reply:
It's not technically adding a dependency that wasn't there before though, right? All supported TFMs include System.Linq.

Reply:
We'll address this in a different PR.